packages/app/src/runner/useRunnerStyle.ts

```diff
@@ -46,8 +46,6 @@ export const useRunnerStyle = ({
   })
 
   const containerHeight = computed(() => {
-    // TODO: in UNIFY-595 the header's contents will be finalized
-    // at narrow widths content will start to wrap
     const autHeaderHeight = 70
```

Comment:
[UNIFY-1340](https://cypress-io.atlassian.net/browse/UNIFY-1340?atlOrigin=eyJpIjoiNTBhYTA0NjRhYzJkNGU2YTg0YWFhZDEzZGYyMzI0Y2MiLCJwIjoiaiJ9) was logged to track this work.